python/mlx/nn/losses.py

```diff
@@ -2,7 +2,16 @@
 
 import mlx.core as mx
 
-
 def cross_entropy(logits: mx.array, targets: mx.array, axis: int = -1):
     score = mx.take_along_axis(logits, targets[..., None], axis).squeeze(-1)
     return mx.logsumexp(logits, axis=axis) - score
+
+def MLE_loss(logits: mx.array, targets: mx.array, axis: int = -1):
```

Comment:
What does `MLE` stand for? This loss looks the same as the `cross_entropy` except it does a mean at the end? I think it would make more sense to just add a reduce option to the `cross_entropy` loss.

Reply:
I like the way PyTorch does reductions, maybe we follow that with the `reduction` argument?